cmd/template/cluster/flag.go

```diff
@@ -28,6 +28,14 @@ const (
 	flagAWSRole             = "role"
 	flagNetworkAZUsageLimit = "az-usage-limit"
 	flagNetworkVPCCidr      = "vpc-cidr"
+	flagClusterType         = "cluster-type"
```

Comment:
are these flags only used for AWS? in that case I think it would make sense to include provider in the var name like below.

Reply:
done